src/test/java/com/puppycrawl/tools/checkstyle/api/AbstractViolationReporterTest.java

```diff
@@ -152,4 +166,17 @@ public int[] getRequiredTokens() {
 
     }
 
+    public static class SetSeverity extends AbstractViolationReporter {
+
+        @Override
+        public void log(int line, String key, Object... args) {
+
+        }
+
+        @Override
+        public void log(int line, int col, String key, Object... args) {
+
+        }
+    }
+
```

Comment:
@romani please see if we can even write tests for this file through input files at all. I have tried writing tests  like the other files in api test directory that use input files but ``checker`` error keeps showing up.